frontend/src/routes/editor/nav/FileNavigation.svelte

```diff
@@ -83,10 +83,12 @@
 		border: none;
 		background: none;
 		color: var(--foreground-0);
-		font-size: larger;
+		font-size: 1rem;
```

Comment:
1rem might already be the default size, make sure that this actually does something

Reply:
switched to inherit